src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -59,7 +59,7 @@ internal static SafeSslHandle AllocateSslContext(SslProtocols protocols, SafeX50
                     throw CreateSslException(SR.net_allocate_ssl_context_failed);
                 }
 
-                Ssl.SetProtocolOptions(innerContext, protocols);
+                Ssl.SetProtocolOptions(innerContext.DangerousGetHandle(), protocols);
```

Comment:
I'll add a comment.  It's safe because we just created the handle, nothing else has a reference to it, we're keeping it rooted via the safe handle and its call to Dispose as part of the using, etc.